### PR TITLE
Fix comment line length

### PR DESCRIPTION
**Pull request by chubin**

### CODE REVIEW

**Comment by chubin** [comment]
@terminalforlife I've fixed length in some cheat sheets. Could you please take a look?

File: sheets/irb [hunk 1]
```diff
@@ -1,5 +1,7 @@
-# interactive ruby. irb is a tool to execute interactively ruby expressions read from stdin
+# irb
+# Interactive ruby. A tool to execute interactively Ruby expressions
+# read from stdin.
 
-# start irb
+# Start irb
```
Comment (terminalforlife) [inline sheets/irb:5]: I have to wonder if this example is even needed. Lol

File: sheets/ncmpcpp [hunk 2]
```diff
@@ -1,6 +1,8 @@
-# ncmpcpp is an mpd client (compatible with mopidy) with a UI very similar to ncmpc,
-# but it provides new useful features such as support for regular expressions for library searches,
-# extended song format, items filtering, the ability to sort playlists, and a local filesystem browser.
+# ncmpcpp
+# An mpd client (compatible with mopidy) with a UI very similar to ncmpc,
+# but it provides new useful features such as support for regular expressions
+# for library searches, extended song format, items filtering, the ability to
+# sort playlists, and a local filesystem browser.
```
Comment (terminalforlife) [inline sheets/ncmpcpp:5]: Noice. I'd still use that trailing '-' after the comma at the end of the line, though.

**Review by terminalforlife — COMMENTED** [review]
Hope this helps. I realise I was probably talking more about grammar more than line length, but I thought maybe you'd find it interesting, as I'm guessing you're not a native English-speaker.

Where are you from, BWT? Russia comes to mind, for some reason.

**Comment by chubin** [comment]
@terminalforlife Thank you for the suggestions. I've fixed everything

**Pull request merged**